src/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,33 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+// const moment = require('moment-timezone');
+
+const getOrderByUserId = async (user_id) => {
+  if (isNaN(user_id)) {
```

Comment:
Be careful with `isNaN`, it only checks if `user_id` is equal to the global constant `NaN`. It does not check that `user_id` is null, undefined, or a string that is not a number. 
You'll need to extend this condition to catch those cases.

I'd suggest combining isNaN with parseInt (in HTTP all values are strings to being with).